modules/conflict_resolver/README.md

```diff
@@ -0,0 +1,68 @@
+# Conflict Resolver
+
+## Purpose
+
+The conflict resolver is intended to allow the resolution of conflicts
+(different data entry for a given field between the "single data
+entry" and "double data entry" variations) for instruments which
+have double data entry enabled.
+
+## Intended Users
+
+The conflict resolver is usually used by data entry staff to resolve
+double data entry conflicts.
+
+Some studies may have protocols which involve supervisors or
+committees deciding how to properly resolve conflicts. For example,
+studies may adopt different policies regarding the personnel that
+should be responsible for conflict resolution (which could be the
+same data entry staff that performed the double data entry or a
+different data entry staff). Such policies are not enforced by
+LORIS.
+
+## Scope
+
+The module only provides data entry conflict resolution and triggers
+the recalculation of fields which are dependent on the data that's
+been changed (ie. score fields, age, etc). It does not do the initial
```

Comment:
I know, a very picky one: "ie." should probably be "i.e."?

Reply:
done